.github/workflows/e2e_tests.yml

```diff
@@ -159,7 +163,7 @@ jobs:
           echo "reports/${{ github.head_ref || github.ref_name }}/${{ github.run_id }}/${{ github.run_attempt }}" > ./tests/artifacts/report-link
           echo "${{ github.event.pull_request.number }}" > ./tests/artifacts/report-pr
 
-      - uses: actions/upload-artifact@v4
+      - uses: actions/upload-artifact@v3
```

Comment:
Is this intentional? https://github.blog/changelog/2024-04-16-deprecation-notice-v3-of-the-artifact-actions/